test/graph/chatChannelFragment.graphql

```diff
@@ -0,0 +1,3 @@
+fragment ChannelName on ChatChannel {
```

Comment:
Why is there a graph and a graphql directory under test?

Reply:
Good question. That shouldn't be.

Reply:
Ah, no that is fine: `graph` is the package that have the tests in and also some graphql files for testing relative imports. `graphql` is for testing resolution from a graphql directory.